packages/babel-jest/README.md

```diff
@@ -20,6 +20,6 @@ To explicitly define `babel-jest` as a transformer for your JavaScript code, map
 
 ```json
 "transform": {
-  "^.+\\.jsx?$": "babel-jest"
+  "^.+\\.(t|j)sx?$": "babel-jest"
```

Comment:
```suggestion
  "^.+\\.[jt]sx?$": "babel-jest"
```

To match the actual default: https://github.com/facebook/jest/blob/f7f787d24ca15ff3949f011c01d933336ba9da4c/packages/jest-config/src/constants.ts#L11

Reply:
also, please update line 19 to mention typescript

Reply:
👍 updated